TMDBApp/TMDBApp/Movie.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  Movie.swift
+//  TMDBApp
+//
+//  Created by Richard Leh on 18/12/2016.
+//  Copyright © 2016 Richard Leh. All rights reserved.
+//
+
+import Foundation
+
+enum ServerTMDBMovie:String{
+    case id = "id"
+    case title = "title"
+    case releaseDate = "release_date"
+    case posterPath = "poster_path"
+    case backdropPath = "backdrop_path"
+    case genreIds = "genre_ids"
+    case overview = "overview"
+}
+
+class Movie{
+    
+    var id:Int?
+    var title:String?
+    var genreIds:[Int]?
+    var posterPath:String?
+    var backdropPath:String?
+    var releaseDate:String?
+    var overview:String?
+    
+    var genres:[Genrer]?
+    
+    convenience init(withDict dictionary:DictionaryData, andGenres genresArr:[Genrer]) {
+        self.init()
+        
+        if let _id = dictionary[ServerTMDBMovie.id.rawValue] as? Int{
+            self.id = _id
```

Comment:
I would change the optional values for some obligate values.
But I can't see anyway to improve this code.